wix-bookings/guides/About Wix Bookings.md

```diff
@@ -4,42 +4,58 @@
+Wix Bookings supports 2 different types of services:
+- Session-based service: Generally a group session that has a set day and time, and takes place regardless of the number of participants that sign up (e.g., a yoga class that takes place every Tuesday at 16:00 or a class that has 8 sessions that take place on Wednesdays at 19:00).
+- Availability-based service: Generally a 1-on-1 or group session that is available for booking, but will only take place if a participant books it (e.g., a barber that is available for appointments Monday through Friday, from 08:00 to 17:00).
+- Both of these types of services can exist within one business - see [Schedules and Sessions Use Cases](https://dev.wix.com/api/rest/wix-bookings/schedules-and-sessions/introduction).
 
 </blockquote>
 
 ![AboutBookingsImage](../../media/BookingsSchedules.png)
 
 ## Terminology
 
+- **Site owner:** The Wix site owner (who has Wix Bookings installed).
+
+- **Resources:** A business asset like a staff member, a room, or equipment needed to provide a service.
+
+- **Service:** A business offering that you can book. For example, a fitness studio may offer a 1-hour yoga class, a 45 minute HIIT training session, and a 30 minute 1-on-1 personal training. Wix Bookings offers 3 types of services:
+
+  + [**Appointment**](https://support.wix.com/en/article/creating-the-right-booking-service-for-your-business#appointments): A specific time slot that a customer can book in a calendar.
+
+  + [**Class**](https://support.wix.com/en/article/creating-the-right-booking-service-for-your-business#classes): A single session or a set of sessions that are offered on specific days and at specific times.
+  + [**Course**](https://support.wix.com/en/article/creating-the-right-booking-service-for-your-business#courses): A multi-session course, or a workshop that starts and ends on a particular day. Customers sign up for the entire course and not one session.
+
+- **Sessions:** A reserved period of time for an appointment, class, or course, or a period of availability for a resource.
+
+- **Schedule:** A collection of sessions and the information needed to calculate availability.
 
+- **Booking:** Information about an order placed by a customer for a service or series of services. Includes details about the time, location, participants, and price.
 
+- **Slot:** An available period of time in a schedule that can be booked by a customer. While this includes existing sessions that are available for booking, it can also represent a period of time that can be booked based on the availability of a resource (e.g., a barber with appointments of 30 minutes each that are open for booking every weekday between 8:00 - 17:00). These slots are calculated by the constraints of the schedule.
 
+- [**Calendar**:](https://support.wix.com/en/article/wix-bookings-about-the-wix-booking-calendar) Where you schedule appointments, classes, and courses.
```

Comment:
this doesn't read like a definition, more like extra information. maybe something like:
a collection of all schedules, appointments, classes, and courses
?